samples/client/petstore/typescript-inversify/api/pet.service.ts

```diff
@@ -36,15 +37,15 @@ export class PetService {
             this.basePath = this.APIConfiguration.basePath;
     }
 
-
     /**
      * Add a new pet to the store
      * 
      * @param body Pet object that needs to be added to the store
      
      */
-
-    public addPet(body: Pet, headers: Headers = {}): Observable<any> {
+    public addPet(body: Pet, observe?: 'body', headers?: Headers): Observable<any>;
```

Comment:
@gualtierim can you find out why the response is `any`? it should be `Pet`, shouldn't it?

Reply:
ah, it's because the `addPet` endpoint does not return anything, so it is ok